solr/solrj/src/java/org/apache/solr/client/solrj/util/Cancellable.java

```diff
@@ -18,9 +18,11 @@
 package org.apache.solr.client.solrj.util;
 
 /**
- * The return type for solrJ asynchronous requests, providing a mechanism whereby callers may
- * request cancellation.
+ * @deprecated Use the async variants that return CompletableFuture.
```

Comment:
This line is supposed to be at the end of the javadoc; isn't supposed to encompass current description.